docs/using-latest-daily.md

```diff
@@ -37,11 +37,11 @@ If you use [Package Source Mapping](https://learn.microsoft.com/en-us/nuget/cons
 To be able to create aspire projects, you will need to install the latest Aspire templates. You can do this by running the following command:
 
 ```sh
-dotnet new install Aspire.ProjectTemplates::9.0.0-rc.1.* --force
+dotnet new install Aspire.ProjectTemplates::* --force
```

Comment:
do we not need to specify a feed here?

Reply:
Feed is above in `Add necessary NuGet feeds`

Reply:
I tried this and it didn't work. I kept getting 8.2.1. I think there has to be come allow pre-release thing here.

Reply:
Did you configure the feed?

Reply:
```suggestion
dotnet new install Aspire.ProjectTemplates::*-* --force
```